.gitignore

```diff
@@ -63,3 +63,5 @@ default.etcd
 # Generated Dockerfile
 # This is automatically generated by Make depending on which image you want to build.
 Dockerfile
+
+vendor/
```

Comment:
Is there any reason we use probably use `vendor/` folder here?

Reply:
Personally, when I need to modify the vendor codes like `/pkg`, I'd rather edit in the `vendor/` instead of `mod`.

Reply:
ok